lib/config/presets/internal/replacements.ts

```diff
@@ -181,6 +182,17 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'eslint-plugin-node-to-maintenance-fork': {
+    description: 'Maintenance fork of `eslint-plugin-node`',
```

Comment:
```suggestion
    description: 'Replace stale `estlint-plugin-node` with a maintained fork: `eslint-plugin-n`.',
```

How about something like this? The description should make it easy to see _what's_ changing and _why_. Also we should end sentences with a full stop (`.`). 😉